### PR TITLE
[SyncWith:zkllvm#510] 510 lpc batching

### PR DESCRIPTION
**Pull request by ETatuzova**

### CODE REVIEW

File: libs/circifier
Comment (CblPOK-git) [inline libs/circifier:1]: I am 100% sure that you haven't made any changes in the circifier lib. If so, it was added mistakenly. 

As I see, it checkouts circifier to the older version of master:
9266caaa758aef44f272c7f814485f4f49496960 is up to date with all commits from 80a8e0f62fa21657c80df72d57a035255a0a4f23.

Looks like you had local old zkllvm + old circifier, then you updated zkllvm, but did not update submodules, and in the end used `"git add ."` and older version of circifier was added into commit. Please remove circifier update and double-check that you did not update other submodules to the older versions, if it happened with circifier then there is non-zero probability that it happened with other submodules

File: CMakeLists.txt [hunk 1]
```diff
@@ -209,4 +209,4 @@ add_custom_target(rslang_dist
                   DEPENDS rslang
                   COMMENT "Building rslang distribution package"
                   USES_TERMINAL
-                  WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/libs/rslang)
+                  WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/libs/rslang)
```
Comment (CblPOK-git) [inline CMakeLists.txt:212]: Half of the files have "empty" changes. Not critical, but also not great

File: libs/blueprint
Comment (CblPOK-git) [inline libs/blueprint:1]: Same thing as in the circifier, f6f31cdec9b2f4a3b18db6ea47f8a8592018aa66 is up to date with all commits from 96efd77999a8e4445c5e27c7274d03321741c8c1

File: libs/stdlib
Comment (CblPOK-git) [inline libs/stdlib:1]: and here
279788538e8b6d4a7a58aa3cdbed9fff9c790e0e is up to date with all commits from 98b0c4e9cd2854e96856ea88d56971c8c80737eb

**Review by CblPOK-git — CHANGES_REQUESTED** [review]

**Review by akokoshn — APPROVED** [review]

**Review by CblPOK-git — APPROVED** [review]

**Pull request merged**